python/lsst/sims/GalSimInterface/galSimPSF.py

```diff
@@ -155,7 +156,49 @@ def __init__(self, fwhm=0.6):
         #for r at half the maximum of the PSF
         alpha = fwhm/2.3835
 
-        gaussian1 = galsim.Gaussian(sigma=alpha)
-        gaussian2 = galsim.Gaussian(sigma=2.0*alpha)
+        rms_pixel_size = pixel_scale*pixel_scale/6.0
+
+        sigma = numpy.sqrt(alpha*alpha - rms_pixel_size)
+        gaussian1 = galsim.Gaussian(sigma=sigma)
+
+        sigma = numpy.sqrt(4.0*alpha*alpha - rms_pixel_size)
+        gaussian2 = galsim.Gaussian(sigma=sigma)
 
         self._cached_psf = 0.909*(gaussian1 + 0.1*gaussian2)
+
+
+class Kolmogorov_and_Gaussian_PSF(PSFbase):
+    """
+    This PSF class is based on David Kirkby's presentation to the DESC Survey Simulations
+    working group on 23 March 2017.
+
+    https://confluence.slac.stanford.edu/pages/viewpage.action?spaceKey=LSSTDESC&title=SSim+2017-03-23
+
+    (you will need a SLAC Confluence account to access that link)
+    """
+
+    def __init__(self, airmass=1.2, rawSeeing=0.7, band='r'):
+        """
+        Parameters
+        ----------
+        airmass
+
+        rawSeeing is the FWHM seeing at zenith at 500 nm in arc seconds
+        (provided by OpSim)
+
+        band is the bandpass of the observation [u,g,r,i,z,y]
+        """
+        # This code was provided by David Kirkby in a private communication
+
+        wlen_eff = dict(u=365.49, g=480.03, r=622.20, i=754.06, z=868.21, y=991.66)[band]
+        FWHMatm = rawSeeing * (wlen_eff / 500.) ** -0.3 * airmass ** 0.6
+        FWHMsys = 0.4 * airmass ** 0.6
```

Comment:
From LSST-20160 eqn (4.2).  Could change 0.4 to sqrt(0.25 ** 2 + 0.30 ** 2 + 0.08 ** 2) for completeness.